server/data/quotes.json

```diff
@@ -315,6 +315,9 @@
     "author": "Ingrid Bengis",
     "sentence": "For me, words are a form of action, capable of influencing change."
   },
+  {"author": "J.R.R. Tolkien",
+  "sentence": "It is wrong to call the language you grow up speaking your native language. It is not. It is your first learnt language."
+  },
```

Comment:
Please format the JSON code. You can use this [tool](https://jsonformatter.curiousconcept.com).

Make sure to [check the linting](https://jsonlint.com).

Reply:
Have run it through the tool, I think it took off an extra space, though, so I re-added that. Should be correct now.